examples/components/MNIST/traininsilo/run.py

```diff
@@ -92,22 +97,69 @@ def load_dataset(self, train_data_dir, test_data_dir):
 
         return train_dataset, test_dataset
 
+    def log_params(self, client, run_id):
+        client.log_param(
+            run_id=run_id, key=f"learning_rate {self._experiment_name}", value=self._lr
+        )
+        client.log_param(
+            run_id=run_id, key=f"epochs {self._experiment_name}", value=self._epochs
+        )
+        client.log_param(
+            run_id=run_id,
+            key=f"batch_size {self._experiment_name}",
+            value=self._batch_size,
+        )
+        client.log_param(
+            run_id=run_id,
+            key=f"loss {self._experiment_name}",
+            value=self.loss_.__class__.__name__,
+        )
+        client.log_param(
+            run_id=run_id,
+            key=f"optimizer {self._experiment_name}",
+            value=self.optimizer_.__class__.__name__,
+        )
+
+    def log_metrics(self, client, run_id, key, value):
+        client.log_metric(
+            run_id=run_id,
+            key=key + f", {self._experiment_name}",
```

Comment:
could we use some programmatically easy naming convention?

for instance: {iteration}/{silo}/{metric} so that in the UI it becomes easy to browse visually, and we could also parse those in mlflow later?

Reply:
Loved this